api/src/test/java/org/openmrs/api/ObsServiceTest.java

```diff
@@ -1459,14 +1459,28 @@ public boolean supportsView(String view) {
 			os.registerHandler("ImageHandler", originalHandler);
 		}
 	}
-
-
+	
 	/**
 	 * @see ObsService#purgeObs(Obs,boolean)
 	 */
 	@Test
-	public void purgeObs_shouldThrowAPIExceptionIfGivenTrueCascade() {
-		assertThrows(APIException.class, () -> Context.getObsService().purgeObs(new Obs(1), true));
+	public void purgeObs_shouldPurgeObsAndChildrenIfGivenTrueCascade() {
+		executeDataSet(INITIAL_OBS_XML);
+		ObsService obsService = Context.getObsService();
+
+		// Obs #2 is a parent with group members (Obs #9, Obs #10) in the test dataset
+		Obs obs = obsService.getObs(2);
+		assertNotNull(obs);
+		assertTrue(obs.hasGroupMembers());
+
+		// Perform purge with cascade=true
```

Comment:
Kindly could we remove this comment? The code is self explanatory, so we can keep it simpler without the extra note please!